src/wallet/wallet.cpp

```diff
@@ -40,12 +40,17 @@
 using namespace std;
 
 CWallet *pwalletMain = nullptr;
+int MAX_FEE_PERCENT_OF_VALUE = 25;
 /** Transaction fee set by the user */
```

Comment:
if I'm sending 100 BCH does this mean I may accidentally send 25 BCH as a fee?  if so seems we should have some kind additional max limiting value?

Reply:
hmm... this was useful in the BTC days but maybe is not so useful anymore. I think it should be set to 1% in mainnet and 25% in regtest/testnet (its set high because tests send dust).

Reply:
I set it to 1% and turn it off for the regtest tests that send dust.